test/raven.test.js

```diff
@@ -364,7 +365,7 @@ describe('globals', function() {
         func: 'lol'
       };
 
-      Raven._globalOptions.includePaths = /^http:\/\/example\.com/;
+      Raven._globalOptions.includePaths = [/^http:\/\/example\.com/];
```

Comment:
Can you leave one as a string instead of an array to make sure that both types work?